integration-tests/reactive-messaging-kafka/pom.xml

```diff
@@ -13,6 +13,12 @@
     <name>Quarkus - Integration Tests - Reactive Messaging - Kafka</name>
     <description>The Kafka Reactive Messaging integration tests module</description>
 
+    <properties>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+        <!-- needs to wait for smallrye-reactive-messaging-health to update MP health to 3.0 -->
+        <enforcer.skip>true</enforcer.skip>
```

Comment:
Is this about `dependencyConvergence`? I'm a bit wary about skipping _all_ enforcer rules like this, e.g. see #15841.
I understand that it's of utmost importance get this branch merged but this should be narrowed ASAP after the merge.

Reply:
I will fix it just after the merge. 
MP Health has nasty breaking changes, so we had to work upstream first. I've all fine now.